test/com/google/enterprise/adaptor/ad/AdAdaptorTest.java

```diff
@@ -1685,43 +1725,7 @@ public long getHighestCommittedUSN() {
         }
       };
     }
-    @Override
-    void getModifiedDocIdsHelper(DocIdPusher pusher)
```

Comment:
What's going on with removing this?

Reply:
I wanted to test full + incremental group pushes (and failed to get it to work). The first part of doing that was to remove this stub.  The stub didn't really matter, since the production code correctly notices that the group definitions have not changed in any of the tests that called getModifiedDocIds().